test/unit/crypto/cryptoKeyFailuresTest.ts

```diff
@@ -76,11 +77,13 @@ if (os.platform() === 'darwin') {
       try {
         await shouldThrow(Crypto.create(), 'Should have thrown an SetCredentialError for Crypto.init()');
       } catch (err) {
-        expect(err.name).to.equal('SetCredentialError');
```

Comment:
oh, this would be a good one to assert `instanceof SfError` so then you get the `actions` typing, etc.